modules/Forum/pages/forum/view_topic.php

```diff
@@ -259,7 +259,7 @@
         ]);
 
         if ($validate->passed()) {
-            $content = Output::getClean(Input::get('content'));
+            $content = Input::get('content');
```

Comment:
Can we verify this has no effect on inserting code snippets which include HTML tags please?

Reply:
Yes, this seems to still work. Editing and viewing works normally.

This is what is stored in the DB: 
```<pre class="language-markup"><code>&lt;p&gt;Hello, I am &lt;strong&gt;HTML&lt;/strong&gt;&lt;/p&gt;</code></pre>```

<img width="1164" alt="Screen Shot 2022-05-02 at 14 27 54" src="https://user-images.githubusercontent.com/26070412/166330549-f3e136c5-0f8b-4889-ab0f-f5e3ab44e7e3.png">